SPECS/libkcapi/libkcapi.spec

```diff
@@ -47,6 +47,10 @@ done                                                             \
   "$lib_path"/libkcapi.so.%{version} || exit 1                   \
 ln -s libkcapi.so.%{version}.hmac                            \\\
   "$lib_path"/fipscheck/libkcapi.so.%{vmajor}.hmac               \
+{ %{sha512hmac} "$lib_path"/libkcapi.so.%{version} || exit 1; }      \\\
```

Comment:
What is this stuff doing?

Reply:
This produces a measurement (libkapi.so.1.hmac) for libkcapi.so.1, which is loaded by kcapi-hasher during initramfs fips verification.